### PR TITLE
feat: added toast component

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after changes.

File: toast.html
Comment (rajaranjanmr) [inline toast.html:84]: remove unnecessary spaces

File: style.css
Comment (rajaranjanmr) [inline style.css:95]: remove unnecessary spaces

**Review by rajaranjanmr — APPROVED** [review]
after changes in code, it can be merged. It seems god to me

File: toast.html
Comment (eakansh125) [reply toast.html:84]: Sure

File: style.css
Comment (eakansh125) [reply style.css:95]: Sure

Comment (eakansh125) [reply style.css:95]: Removed

**Pull request merged**
